www/src/data/features/comparison-options.json

```diff
@@ -0,0 +1,42 @@
+{
+  "jamstack": [
+    {
+      "key": "nextjs",
+      "display": "Next.js",
+      "nodeField": "Nextjs",
+      "description": "Next.js is a free and open source web application framework based on React.js, Node.js, webpack and Babel.js for building server-side rendered and/or static web applications using React."
+    },
+    {
+      "key": "jekyll",
+      "display": "Jekyll",
+      "nodeField": "Jekyll",
+      "description": "Jekyll is a simple, blog-aware, static site generator for personal, project, or organization sites. Written in Ruby by Tom Preston-Werner, GitHub's co-founder, it is distributed under the open source MIT license."
+    },
+    {
+      "key": "hugo",
+      "display": "Hugo",
+      "nodeField": "Hugo",
+      "description": "Hugo is a static site generator written in Go. Hugo takes data files, i18n bundles, configuration, templates for layouts, static files, and content written in Markdown and renders a static website. It is an open source project licensed under the Apache License 2.0."
+    },
+    {
+      "key": "nuxtjs",
+      "display": "Nuxt.js",
+      "nodeField": "Nuxtjs",
+      "description": "Next.js is a free and open source web application framework based on Vue.js, Node.js, webpack and Babel.js for building server-side rendered and/or static web applications using Vue."
```

Comment:
Typo here

Reply:
```suggestion
      "description": "Nuxt.js is a free and open source web application framework based on Vue.js, Node.js, webpack and Babel.js for building server-side rendered and/or static web applications using Vue."
```

Reply:
Thanks! GitHub on mobile is just inferior